### ISSUE TITLE
unable to get the cifar10 accuracy

### ISSUE THREAD

**Issue by Dongshengjiang** [leaderj1001/BottleneckTransformers#7]
Hi, I just change your code with ResNet50(num_classes=10, resolution=(224, 224)), which end with a lower accuaracy of 90.15%. do you have other changes to get the 9511%?

**Comment by leaderj1001** [comment]
When I try training, I use 32x32 resolution.
Could you plz try again :)
Thanks

**Comment by yutinyang** [comment]
When I use ResNet50(num_classes=10, resolution=(32, 32)) , remove the max-pooling for imagenet, and change self.conv1 for CIFAR10 , I just get a precision 89% on CIFAR10.

**Comment by baileyyeah0326** [comment]
I just tried ResNet50(num_classes=10, resolution=(32, 32)) , remove the max-pooling for imagenet, and change self.conv1 for CIFAR10 to nn.Conv2d(3, 64, kernel_size=3, stride=1, padding=1, bias=False), but I got gradient explosion. Did you make any other changes?
